esp-hal-common/devices/esp32h2.toml

```diff
@@ -57,6 +57,8 @@ peripherals = [
     "uart1",
     # "uhci0",
     "usb_device",
+    "assist_debug_sp_monitor",
+    "assist_debug_region_monitor",
```

Comment:
Sorry to nitpick, can you please move these down into the next section (under the comment below)